VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
+		\end{itemize}
+	\item [\null] -RET:  Absence of a retrotransposon present in the reference
+	\end{itemize}
+\item [\null] TRA:  Translocation
+\item [\null] STR:  Short tandem repeat variation
+\item [\null] INV:  Inversion relative to the reference
+\item [\null] DUP:  Duplication
+	\begin{itemize}
+	\item [\null] TANDUP:  Tandem duplication (one of Direct, Inverted-upstream, or Inverted-downstream)
+	\item [\null] DISDUP:  Dispersed duplication
+	\end{itemize}
+\item [\null] DELINS:  Deletion-insertion (replacement of reference sequence with sequence of different content or length)
+\item [\null] OTHER:  Complex variant not covered elsewhere
+	\begin{itemize}
+	\item [\null] DEL\_DUP:  Deletion + Duplication
+	\item [\null] DEL\_INV:  Deletion + Inversion
+	\item [\null] DUP\_INV:  Duplication + Inversion
+	\item [\null] DEL\_DUP\_INV:  Deletion + Duplication + Inversion
+	\end{itemize}
+\item [\null] CNV:  Copy number variation\vspace{0.3cm}
 \end{itemize}
-
+TRA (Translocation):  A VCF translocation record describes the juxtaposition of two DNA segments from disparate genomic locations (from the same or from different chromosomes). It must include two genomic coordinates, each accompanied by an orientation (see STRAND below). A VCF translocation record only describes the immediate molecular context of the novel adjacency generated by the translocation event. It should not be confused with the traditional cytogenetic notion of translocation, which is understood to imply that the remainder of the chromosome arms beyond the specified breakpoints are also involved.\vspace{0.3cm}\\
+\setlength{\parindent}{0.3cm}
+\indent In a TRA VCF record:\vspace{0.3cm}\\
+\indent CHROM = chromosome of the first breakpoint\\
+\indent POS = coordinate of the first breakpoint\\
+\indent CHR2 = chromosome of 2nd breakpoint\\
+\indent END = coordinate of 2nd breakpoint\\
+\indent STRAND = (head-tail, tail-head, head-head, tail-tail)\\
+\indent REF = reference base at position CHROM:POS\\
+\indent ALT = reference base at position CHR2:END\vspace{0.3cm}\\
+STR (Short tandem repeat variation, simple sequence repeat, microsatellite): Short tandem repeat variants may be represented in VCF either as structural variants or as ordinary indels (see Section 5.3 for examples). The \textbf{Pindel} software package, for example, treats STRs as standard variants, while \textbf{lobSTR} treats them as structural variants. When treated as structural variants:\vspace{0.3cm}\\
+\indent RPA = repeats per allele\\
+\indent END = end coordinate of the STR\\
+\indent MOTIF = the canonicalized STR repeat motif\\
+\indent NS = number of samples with data\\
+\indent REF = the reference copy number\\
+\indent RL = total length of the STR tract in the reference\\
+\indent RU = repeat motif as it appears on the forward strand in the reference\vspace{0.3cm}\\
+\indent More information on representing STRs in VCF can be found at http://lobstr.teamerlich.org/file-formats.html\vspace{0.3cm}\\
```

Comment:
@SHuang-Broad 
Agreed – the reference was intended more as a citation / acknowledgment than as a resource for more information – though I agree the spec needs clarification, if the term is kept.
That leaves us with two options:
1. Provide enough detail in the spec, including an example, so the reference to lobSTR is not needed
2. Remove STR as a SVTYPE (or SVSUBTYPE / SVSUB) since STRs can be handled by VCF as non-SV records.
Right now I am leaning toward removing STR from the PR.
Thoughts?

Reply:
I prefer the 2nd, so to keep the spec as short as possible.